website/landing/components/common/Section.tsx

```diff
@@ -0,0 +1,49 @@
+import { styled } from "../../stitches.config";
```

Comment:
This layout repeats itself on every section of the landing, I've extracted it here as a starting point and will update the other sections in the next PRs.